wire-protoc-compatibility-tests/src/test/java/com/squareup/wire/Proto3WireProtocCompatibilityTests.kt

```diff
@@ -264,4 +257,253 @@ class Proto3WireProtocCompatibilityTests {
+        rep_float = list(122.0F),
+        rep_double = list(123.0),
+        rep_string = list("124"),
+        rep_bytes = list(ByteString.of(123, 125)),
+        rep_nested_enum = list(AllTypes.NestedEnum.A),
+        rep_nested_message = list(AllTypes.NestedMessage(a = 999)),
+        pack_int32 = list(111),
+        pack_uint32 = list(112),
+        pack_sint32 = list(113),
+        pack_fixed32 = list(114),
+        pack_sfixed32 = list(115),
+        pack_int64 = list(116L),
+        pack_uint64 = list(117L),
+        pack_sint64 = list(118L),
+        pack_fixed64 = list(119L),
+        pack_sfixed64 = list(120L),
+        pack_bool = list(true),
+        pack_float = list(122.0F),
+        pack_double = list(123.0),
+        pack_nested_enum = list(AllTypes.NestedEnum.A),
+        map_int32_int32 = mapOf(1 to 2),
+        map_string_string = mapOf("key" to "value"),
+        map_string_message = mapOf("message" to AllTypes.NestedMessage(1)),
+        map_string_enum = mapOf("enum" to AllTypes.NestedEnum.A)
+    )
+
+    private val DEFAULT_ALL_TYPES_JSON = """{
+        |"int32":111,
+        |"uint32":112,
+        |"sint32":113,
+        |"fixed32":114,
+        |"sfixed32":115,
+        |"int64":"116",
+        |"uint64":"117",
+        |"sint64":"118",
+        |"fixed64":"119",
+        |"sfixed64":"120",
+        |"bool":true,
+        |"float":122.0,
+        |"double":123.0,
+        |"string":"124",
+        |"bytes":"e30=",
+        |"nestedEnum":"A",
+        |"nestedMessage":{"a":999},
+        |"repInt32":[111,111],
+        |"repUint32":[112,112],
+        |"repSint32":[113,113],
+        |"repFixed32":[114,114],
+        |"repSfixed32":[115,115],
+        |"repInt64":["116","116"],
+        |"repUint64":["117","117"],
+        |"repSint64":["118","118"],
+        |"repFixed64":["119","119"],
+        |"repSfixed64":["120","120"],
+        |"repBool":[true,true],
+        |"repFloat":[122.0,122.0],
+        |"repDouble":[123.0,123.0],
+        |"repString":["124", "124"],
+        |"repBytes":["e30=", "e30="],
+        |"repNestedEnum":["A", "A"],
+        |"repNestedMessage":[{"a":999},{"a":999}],
+        |"packInt32":[111,111],
+        |"packUint32":[112,112],
+        |"packSint32":[113,113],
+        |"packFixed32":[114,114],
+        |"packSfixed32":[115,115],
+        |"packInt64":["116","116"],
+        |"packUint64":["117","117"],
+        |"packSint64":["118","118"],
+        |"packFixed64":["119","119"],
+        |"packSfixed64":["120","120"],
+        |"packBool":[true,true],
+        |"packFloat":[122.0,122.0],
+        |"packDouble":[123.0,123.0],
+        |"packNestedEnum":["A", "A"],
+        |"mapInt32Int32":{"1":2},
+        |"mapStringString":{"key":"value"},
+        |"mapStringMessage":{"message":{"a":1}},
+        |"mapStringEnum":{"enum":"A"}
+        |}""".trimMargin()
+
+    private const val IDENTITY_ALL_TYPES_JSON = "{}"
+
+    private fun <T : kotlin.Any> list(t: T): List<T> {
```

Comment:
why?

Reply:
A shortcut to avoid writing twice all data for repeated fields, and thanks to that, statements are mostly one liners.